da/lazyledger/lazyledger.go

```diff
@@ -10,8 +10,13 @@ import (
 	"github.com/pelletier/go-toml"
 	"google.golang.org/grpc"
 
+	"github.com/cosmos/cosmos-sdk/client"
```

Comment:
will this require another replace directive in go.mod?

Reply:
Oh, NVM. It's already there:

https://github.com/lazyledger/optimint/blob/15d5cc4ee04bcfa1c3e0a70085c67d13b0637de5/go.mod#L28-L31

Reply:
I'm not 100% sure yet, but I don't think so. We should avoid that if possible, correct?

Reply:
Nah, all good. I thought `github.com/cosmos/cosmos-sdk` was introduced as a new dependency. That wasn't the case. It was already there. I think @tzdybal is using a different branch of our fork of the SDK. That fork is actually only there to swap the node in the SDK as far as I remember. 
BTW, @tzdybal given that this should be more or less the only change there, it should be easier to keep the SDK on that branch up to date. After the milestone is done, we should update that branch at least (and maybe the rest of our stack as well).

@evan-forbes as mentioned in today's call, I would revisit this PR from the perspective if part the APIs should not instead be part of the lazyledger-app instead. So that the logic to sign etc is not part of this repo. Not 100% sure about this either tbh but it seems like some of the details here should instead be part of the app.

Reply:
I like that idea a lot. [#92](https://github.com/lazyledger/lazyledger-app/issues/92)